src/components/views/menu/menu.jsx

```diff
@@ -78,6 +86,10 @@ export default class Menu extends React.Component {
          */
         pointing: React.PropTypes.bool,
         /**
+         * A menu can be formatted to float left
+         */
+        left: React.PropTypes.bool,
+        /**
          * A menu can be formatted to float right
```

Comment:
Yeah, `left floated` does not exist, just `left` or `right` classes (https://github.com/Semantic-Org/UI-Menu/blob/master/menu.css#L480), fixing it